tests/stress/test_stress.py

```
@@ -99,6 +100,36 @@ def test_should_load_all_markers_and_measure_performance(self, page: Page, perfo
+            for i in range(max_clicks):
+                if individual_markers.count() > 0:
+                    break
+                clusters.first.click()
+                print(f"Click {i + 1}: expanding cluster...")
+                expect(page.locator(".leaflet-marker-icon").first).to_be_visible(
+                    timeout=MARKER_LOAD_TIMEOUT
+                )
+            else:
+                raise AssertionError(
+                    f"No individual markers appeared after {max_clicks} cluster clicks"
+                )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)